frame/bridge/ethereum/issuing/contract/src/lib.rs

```diff
@@ -0,0 +1,379 @@
+// This file is part of Darwinia.
+//
+// Copyright (C) 2018-2021 Darwinia Network
+// SPDX-License-Identifier: GPL-3.0
+//
+// Darwinia is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Darwinia is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Darwinia. If not, see <https://www.gnu.org/licenses/>.
+
+//! Prototype module for cross chain assets issuing.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[macro_use]
+extern crate alloc;
+
+use ethabi::{
+	param_type::ParamType, token::Token, Bytes, Error, EventParam, Function, Param,
+	RawLog, Result as AbiResult,
+};
+use ethereum_types::{Address as EthereumAddress, H160, H256, U256};
+
+pub use ethabi::{Event, Log};
+use sp_std::vec::Vec;
+
+pub struct Abi;
+
+pub struct Address([u8; 20]);
```

Comment:
Change to `sp_core::H160` or `ethereum_type::H160` also works?

Reply:
I can see the `dvm` and other pallets have the same version of this type in the latest version.
We can try to change it to H160 directly now.